plugins/amd-gpu/fu-amd-gpu-device.c

```diff
@@ -104,6 +115,32 @@ fu_amd_gpu_device_probe(FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_amd_gpu_device_set_marketing_name(FuDevice *device, GError **error)
+{
+	g_autoptr(FuDeviceLocker) locker = fu_device_locker_new(device, error);
```

Comment:
I'm confused why this is needed -- the base device should already be open when setup() is called.

Reply:
Oh; you're right!  I fixed.  I also made it non-fatal if libdrm failed.  If it does we at least have the name from udev to go on.